mockgen/internal/tests/empty_interface/input.go

```diff
@@ -2,4 +2,4 @@ package empty_interface
 
 //go:generate mockgen -package empty_interface -destination mock.go -source input.go
 
-type Empty interface{} // migrating interface{} -> any does not resolve to an interface type.
+type Empty any // migrating interface{} -> any does not resolve to an interface type.
```

Comment:
this comment (and one above) doesn't seme to hold anymore? But also I don't understand what this means?